rotkehlchen/exchanges/bit2me.py

```diff
@@ -0,0 +1,927 @@
+        """Bit2Me does not provide margin positions; return empty list."""
+        return []
+
+    def query_online_history_events(
+        self,
+        start_ts: Timestamp,
+        end_ts: Timestamp,
+        force_refresh: bool = False,  # pylint: disable=unused-argument
+    ) -> tuple['Sequence[HistoryBaseEntry]', Timestamp]:
+        """Query all history events from Bit2me (trades + deposits/withdrawals + brokerage).
+
+        This method combines trade history, asset movements (deposits/withdrawals),
+        and brokerage transactions (purchases/sales) into a single list of history
+        events that will be stored in the database.
+        """
+        events: list[HistoryBaseEntry] = []
+
+        # Get deposits/withdrawals from /v2/wallet/transaction
+        movements = self.query_online_deposits_withdrawals(start_ts, end_ts)
+        events.extend(movements)
+
+        # Get brokerage trades from /v2/wallet/transaction (type=transfer, subtype=purchase)
+        brokerage_trades = self._query_brokerage_trades(start_ts, end_ts)
+        events.extend(brokerage_trades)
+
+        # Get exchange trades from /v1/trading/trade
+        trades, _ = self.query_online_trade_history(start_ts, end_ts)
+        events.extend(trades)
+
+        return events, end_ts
+
+    @protect_with_lock()
+    @cache_response_timewise()
+    def query_balances(self) -> ExchangeQueryBalances:
+        """Query Bit2me balances for all assets."""
+        try:
+            response = self._api_query('balances')
+        except RemoteError as e:
+            msg = f'Bit2me API request failed. {e!s}'
+            log.error(msg)
+            return None, msg
+
+        try:
+            response_data = jsonloads_list(response.text)
+        except JSONDecodeError as e:
+            msg = f'Bit2me returned invalid JSON response. {e!s}'
+            log.error(msg)
+            return None, msg
+
+        # Bit2me API returns a list of balances
+        # Each item has: currency, balance, blockedBalance, blockedOutputBalance
+        # response_data is already validated as a list by jsonloads_list
+
+        assets_balance: dict[AssetWithOracles, Balance] = {}
+
+        for balance_entry in response_data:
+            try:
+                # Get asset symbol (API field is 'currency')
+                asset_symbol = balance_entry.get('currency')
+                if not asset_symbol:
+                    log.warning(f'Bit2me balance entry missing currency symbol: {balance_entry}')
+                    continue
+
+                # Calculate total balance (available + blocked amounts)
+                balance_amount = deserialize_fval(
+                    balance_entry.get('balance', '0'),
+                    'balance',
+                    'bit2me',
+                )
+                blocked_balance = deserialize_fval(
+                    balance_entry.get('blockedBalance', '0'),
+                    'blockedBalance',
+                    'bit2me',
```

Comment:
when breaking the arguments of a function use the name of the variable

```suggestion
                blocked_balance = deserialize_fval(
                    value=balance_entry.get('blockedBalance', '0'),
                    name='blockedBalance',
                    location='bit2me',
```

do it also in other places of this file